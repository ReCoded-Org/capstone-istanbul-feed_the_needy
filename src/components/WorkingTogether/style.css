.workingTogether {
  display: flex;
  margin: 2rem 0;
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
}

.workingTogetherWrittenContent {
  margin-left: 3rem;
  width: 55%;
}

.workingTogetherText {
  height: 12.5rem;
  overflow: scroll;
  max-width: 100%;
}

@media screen and (max-width: 576px) {
  .workingTogether {
    flex-direction: column;
  }

  .workingTogetherWrittenContent {
    margin: 1rem 0 0 0;
    width: 100%;
  }

  .workingTogetherVideo {
    width: 100%;
  }
}
