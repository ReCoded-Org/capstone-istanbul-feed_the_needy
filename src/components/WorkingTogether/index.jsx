import React from "react";
import { Typography } from "antd";
import "./style.css";
import { useTranslation } from "react-i18next";

const { Title, Text } = Typography;

const WorkingTogether = () => {
  const { t } = useTranslation();

  return (
    <div className="workingTogether">
      <iframe
        className="workingTogetherVideo"
        title="Zero Hunger"
        width="50%"
        height="auto"
        src="https://www.youtube.com/embed/iteCytv0RqY"
        frameBorder="0"
        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
        allowFullScreen
      />
      <div className="workingTogetherWrittenContent">
        <Title>{t("workingTogether.title")}</Title>
        <div className="workingTogetherText">
          <Text>{t("workingTogether.text")}</Text>
        </div>
      </div>
    </div>
  );
};

export default WorkingTogether;
