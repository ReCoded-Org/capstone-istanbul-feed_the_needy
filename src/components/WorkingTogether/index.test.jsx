import React from "react";
import renderer from "react-test-renderer";
import WorkingTogether from "./index";

jest.mock("react-i18next", () => ({
  useTranslation: () => ({ t: (key) => key }),
}));

it("Working together renders correctly", () => {
  const tree = renderer.create(<WorkingTogether />).toJSON();
  expect(tree).toMatchSnapshot();
});
