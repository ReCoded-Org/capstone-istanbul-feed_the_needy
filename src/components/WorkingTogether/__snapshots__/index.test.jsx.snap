// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Working together renders correctly 1`] = `
<div
  className="workingTogether"
>
  <iframe
    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
    allowFullScreen={true}
    className="workingTogetherVideo"
    frameBorder="0"
    height="auto"
    src="https://www.youtube.com/embed/iteCytv0RqY"
    title="Zero Hunger"
    width="50%"
  />
  <div
    className="workingTogetherWrittenContent"
  >
    <h1
      className="ant-typography"
      style={
        Object {
          "WebkitLineClamp": null,
        }
      }
    >
      workingTogether.title
    </h1>
    <div
      className="workingTogetherText"
    >
      <span
        className="ant-typography"
        style={
          Object {
            "WebkitLineClamp": null,
          }
        }
      >
        workingTogether.text
      </span>
    </div>
  </div>
</div>
`;
